src/Curators/DBALNewsArticleRepository.php

```diff
@@ -51,6 +51,39 @@ public function getAll(): NewsArticles
         return $this->createNewsArticles($newsArticleRows);
     }
 
+    public function create(NewsArticle $newsArticle): void
+    {
+        $queryBuilder = $this->connection->createQueryBuilder();
```

Comment:
Isn't there an `->insert()` method on DBAL connections that can make this implementation a lot shorter?

Reply:
There is indeed a shorthand `insert` on the `Connection` class but that is not escaped and are not safe for user input. So I always stick with the `QueryBuilder` methods and only use the shorthand ones inside tests.

See https://github.com/doctrine/dbal/blob/v2.9.3/lib/Doctrine/DBAL/Connection.php#L745